packages/u/usysconf-epoch/files/epoch.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+STATE_DIR="${STATE_DIR:=/var/solus/usr-merge}"
+MERGE_FLAG_FILE="${MERGE_FLAG_FILE:=${STATE_DIR}/merge-complete}"
+
+OLD_REPO="https://cdn.getsol.us/repo/shannon/eopkg-index.xml.xz"
+NEW_REPO="https://cdn.getsol.us/repo/shannon/eopkg-index.xml"
```

Comment:
Would it make sense to use .gz here as a stop-gap measure until .zst support in eopkg4 is ready?

Reply:
I've already fixed the issue in eopkg (https://github.com/getsolus/eopkg/pull/151), so it just needs a release/update :slightly_smiling_face:

Reply:
Oh, and the URL is the same because we haven't decided on a new one yet!

Reply:
On the _assumption_ that you can call the actual directories you make ferryd sync from and to pretty much anything (caddy supports suitably named symlinks pointing elsewhere on the fs for the old repo version), how about this:

- `https://cdn.getsol.us/repo/unstable-v2/eopkg-index.xml.xz` ?
- `https://cdn.getsol.us/repo/shannon-v2/eopkg-index.xml.xz` ?

Then you can increment the `-vN` for subsequent epoch bumps if necessary, leaving the previous route in place as a free-standing, pruned pkgdir under /srv somewhere?